server/users.js

```diff
@@ -544,6 +545,16 @@ class User extends Chat.MessageContext {
 		Users.add(this);
 	}
 
+	/**
+	 * @param {UserSettings} [settings]
```

Comment:
`settings` isn't an optional parameter though. You shouldn't be running this function with no parameters. It should probably be something like 
```suggestion
	 * @param {UserSettings?}
```

Reply:
Ugh. Of course you're missing `settings` now

Reply:
my bad